build_with_version.cmake

```
@@ -5,7 +5,6 @@ execute_process(COMMAND bash "-c" "git rev-parse HEAD | tr -d '\n'"
 set(COMPILER ${CMAKE_ARGV3})
 set(PROJECT_VERSION ${CMAKE_ARGV4})
 set(OUTPUT_NAME ${CMAKE_ARGV5})
-set(BUILD_SOURCE ${CMAKE_ARGV6})
 set(INJECT_PATH ${CMAKE_ARGV7})
 set(TRIMPATH ${CMAKE_ARGV8})
```

```suggestion
set(INJECT_PATH ${CMAKE_ARGV6})
set(TRIMPATH ${CMAKE_ARGV7})
```
These probably need to be adjusted if we remove argv[6]

---

good catch!